### PR TITLE
Add performance treshold to L3 tests

### PR DESCRIPTION
**Pull request by JanuszL**
Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

File: docs/examples/pytorch/resnet50/main.py [hunk 1]
```diff
@@ -282,7 +285,8 @@ def main():
             }, is_best)
             if epoch == args.epochs - 1:
                 print('##Top-1 {0}\n'
-                      '##Top-5 {1}'.format(prec1, prec5))
+                      '##Top-5 {1}\n'
+                      '##Perf  {2}'.format(prec1, prec5, args.world_size * args.batch_size / total_time.avg))
```
Comment (jantonguirao) [inline docs/examples/pytorch/resnet50/main.py]: knitpick: `args.world_size * args.batch_size` could be extracted to separate constant

File: qa/L3_RN50_convergence/test_mxnet.sh [hunk 2]
```diff
@@ -22,4 +23,9 @@ if [[ `echo "$best $threshold" | awk '{ print ($1 >= $2) ? "1" : "0" }'` -eq "0"
     exit -1
 fi
 
+if [[ `echo "$mean $min_perf" | awk '{ print ($1 >= $2) ? "1" : "0" }'` -eq "0" ]]; then
```
Comment (jantonguirao) [inline qa/L3_RN50_convergence/test_mxnet.sh]: Perhaps less complicated
```
if (( $(echo "$mean < $min_perf" | bc -l) )); then
    echo "perf = $mean; TEST FAILED"
    exit -1
fi
```

File: qa/L3_RN50_convergence/test_tensorflow.sh [hunk 3]
```diff
@@ -53,13 +54,15 @@ fi
 
 TOP1_RESULT=$(echo "$TOP1 $MIN_TOP1" | awk '{if ($1>=$2) {print "OK"} else { print "FAIL" }}')
 TOP5_RESULT=$(echo "$TOP5 $MIN_TOP5" | awk '{if ($1>=$2) {print "OK"} else { print "FAIL" }}')
+PERF_RESULT=$(echo "$PERF $MIN_PERF" | awk '{if ($1>=$2) {print "OK"} else { print "FAIL" }}')
 
 echo
 printf "TOP-1 Accuracy: %.2f%% (expect at least %f%%) %s\n" $TOP1 $MIN_TOP1 $TOP1_RESULT
 printf "TOP-5 Accuracy: %.2f%% (expect at least %f%%) %s\n" $TOP5 $MIN_TOP5 $TOP5_RESULT
-printf "mean speed = $mean samples/sec\n"
+printf "TOP-5 Accuracy: %.2f%% (expect at least %f%%) %s\n" $PERF $MIN_PERF $PERF_RESULT
```
Comment (jantonguirao) [inline qa/L3_RN50_convergence/test_tensorflow.sh]:
```suggestion
printf "Average perf: %.2f%% (expect at least %f%%) %s\n" $PERF $MIN_PERF $PERF_RESULT
```

File: qa/L3_RN50_convergence/test_tensorflow.sh [hunk 4]
```diff
@@ -53,13 +54,15 @@ fi
...
+printf "mean speed = $PERF samples/sec\n"
 
-if [[ "$TOP1_RESULT" == "OK" && "$TOP5_RESULT" == "OK" ]]; then
+if [[ "$TOP1_RESULT" == "OK" && "$TOP5_RESULT" == "OK" && "$TOP5_RESULT" == "OK" ]]; then
```
Comment (jantonguirao) [inline qa/L3_RN50_convergence/test_tensorflow.sh]:
```suggestion
if [[ "$TOP1_RESULT" == "OK" && "$TOP5_RESULT" == "OK" && "$PERF_RESULT" == "OK" ]]; then
```

**Pull request merged**